### PR TITLE
Changes to fit to P14, including post-test fixes

### PR DESCRIPTION
**Pull request by rhfogh**
Changed from calling collect_hwobj to calling underlying hwobj
dditional GPhL bug fix

### CODE REVIEW

**Comment by IvarsKarpics** [comment]
Hi @rhfogh , could you please resolve the conflict. I would suggest to make a copy of GphlWorkflowQueueEntry, checkout out the queue_entry from master and paste GphlWorkflowQueueEntry into HardwareObjects/Gphl/GphlQueueEntry.py

**Comment by rhfogh** [comment]
@IvarsKarpics 
In progress.

**Comment by rhfogh** [comment]
All post-test bug fixes added. No longet WIP, but ready to review

File: HardwareObjects/abstract/AbstractCollect.py [hunk 1]
```diff
@@ -504,6 +504,9 @@ def get_beam_centre(self):
         else:
             return None, None
 
+    def get_beam_centre_pix(self):
+        return self.detector_hwobj.get_beam_centre_pix()
```
Comment (IvarsKarpics) [inline HardwareObjects/abstract/AbstractCollect.py]: Maybe you can add get_beam_centre_pix to AbstractDetector?

File: ConvertUtils.py [hunk 2]
```diff
@@ -16,24 +16,30 @@
...
 
 # Constants
 
-# Conversion from kEv to A, wavelength = H_OVER_E/energy
-H_OVER_E = 12.3984
-
 
-# Enumerations:
+# 'Borrowed' from six, pending installation as a dependency
+PY2 = sys.version_info[0] == 2
```
Comment (IvarsKarpics) [inline ConvertUtils.py]: Looks like PY2 is not used and PY3 is used just once. Maybe remove them?

File: HardwareObjects/abstract/AbstractCollect.py (hunk 1 above)
Comment (rhfogh) [reply HardwareObjects/abstract/AbstractCollect.py]: Not qure how to do this. Some beamlines definitely use get_beam_centre to return values in pixels, see e.g. this from ALBACollect:
        beamx, beamy = self.detector_hwobj.get_beam_centre()
        self.image_headers["Beam_xy"] = "(%.2f, %.2f) pixels" % (beamx, beamy)

Or this, from BIOMAXCollect:
        beam_centre_x, beam_centre_y = (
            self.get_beam_centre()
        )  # self.get_beam_centre_pixel() # returns pixel
        config["BeamCenterX"] = beam_centre_x  # unit, should be pixel for master file
        config["BeamCenterY"] = beam_centre_y

Or this, from DetectorMockup:
    def get_beam_centre(self):
        """Get approx detector centre (default to Pilatus values)"""
        xval = self.getProperty('width', 2463)/2. + 0.4
        yval = self.getProperty('height', 2527)/2. + 0.4
        return  xval, yval

Should we not either leave things unchanged, or make a PR where we fix everything in a consistent manner, which people can then discuss?

File: ConvertUtils.py (hunk 2 above)
Comment (rhfogh) [reply ConvertUtils.py]: Checked in proposed fix (The new PYVERSION might be useful to have)

File: HardwareObjects/abstract/AbstractCollect.py (hunk 1 above)
Comment (IvarsKarpics) [reply HardwareObjects/abstract/AbstractCollect.py]: Ok, lets leave it as it is and agree on it later.

**Pull request merged**